schema/swagger.yml

```diff
@@ -42,12 +46,12 @@ tags:
   description: Node related API calls
 - name: Peers
   description: Peers related API Calls
-- name: Queue
-  description: Queue related API calls
 - name: Signatures
   description: Signatures related API calls
 - name: Transactions
   description: Transactions related API calls
+- name: Voters
+  description: Votes related API calls
 - name: Votes
   description: Votes related API calls
 - name: Private
```

Comment:
Suggest to change this `Private` to `Forging` or something related. Because this tag applies to endpoint restrict with `config.forging.access.whiteList`

Reply:
I decided to remmove the `Private` tag completely, as the API calls are now grouped by their endpoints.